web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-config-topics.adoc

```diff
@@ -0,0 +1,92 @@
+
+In the following example, you declare and reference an HTTP Requester configuration with HTTP Basic authentication in the *Transport Configuration* parameter of the Web Service Consumer Connector configuration, so the connector dispatches each requests using the advanced configuration:
+
+. In Studio, open the *Global Element Properties* window of the Web Service Consumer Connector.
+. On the *Transport* tab, for *Transport Configuration* select `HTTP Transport Configuration`. +
+The *Requester Config* field automatically populates `HTTP_Request_configuration`.
+. Click *OK*
+
+In Studio, configuring Custom HTTP Transport looks as follows:
+
+.Configure Custom HTTP Transport
+image::web-service-consumer-configure-transport.png[Configure Custom HTTP Transport]
+
+In the *Configuration XML* tab, the reference of the HTTP Requester configuration `<http:request-config name="basic-auth">` is under the Web Service Consumer configuration `<wsc:custom-transport-configuration>`, and looks as follows:
+
+[source,xml,linenums]
+----
+<http:request-config name="basic-auth">
+    <http:request-connection host="www.exampleplace.com" port="8182" protocol="HTTP">
+        <http:authentication>
+            <http:basic-authentication username="admin" password="textpassword"/>
+        </http:authentication>
+    </http:request-connection>
+</http:request-config>
+
+<wsc:config name="with-custom-transport">
+    <wsc:connection wsdlLocation="http://www.host.com?WSDL" service="Service" port="Port">
+        <wsc:custom-transport-configuration>
+            <wsc:http-transport-configuration requesterConfig="basic-auth"/>
+        </wsc:custom-transport-configuration>
+    </wsc:connection>
+</wsc:config>
+----
+
+
+
+== Configure Web Service Security
+
+The WS-Security protocol specifies how you can enforce integrity and confidentiality on messages and enables the communication of various security token formats. Its main focus is the use of XML Signature and XML Encryption to provide end-to-end security.
+
+WS-Security describes three main mechanisms:
+
+* How to sign SOAP messages to assure integrity, signed messages also provide nonrepudiation
+* How to encrypt SOAP messages to assure confidentiality
+* How to attach security tokens to ascertain the sender's identity
+
+The Web Service Consumer Connector supports the following mechanisms:
+
+* Timestamp
+* Signature or Signature Verification
+* Username
+* Encryption or Decryption
+
+In the following example, you configure a username token security strategy as Web Service Security for the connector:
+
+. In Studio, open the *Global Element Properties* window of the Web Service Consumer Connector.
+. On the *Security* tab, for *Username token security strategy* select `Edit inline`.
```

Comment:
Important: We are no longer using RELEASE in the .pom file example because one of the Gastons told us it was a security issue. If you look at the latest template for the XML topic, you will see different wording for the version.